### DISCUSSION TITLE
How to disconnect user's pptp or l2tp connections ? Can not find ppp's PID .

### DISCUSSION THREAD

**Discussion by carasha** [accel-ppp/accel-ppp#170] [General]
My Server OS : CentOS 7

Original pptpd and xl2tpd , when a user connected , linux will create PID for each ppp connections , so i can force to disconnect the ppp connecttion depends on my conditions ( for example : a user who is expired ) .

In pptpd or l2tpd , i can disconnect user according to it's ppp connection's PID . ( i can found in /var/run/ppp1.pid , and kill -s 9 PID )

But accel-ppp seems won't create PID for each ppp connections ( pptp or l2tp ) , so how to disconnect user ?? any suggestions ?

**Comment by svlobanov** [comment]
Hi @carasha , you should use accel-cmd to terminate session, run accel-cmd help, then find how to terminate the sessions
